env.tsa.sh

```diff
@@ -46,18 +45,18 @@ setupDefaults()
 {
     # available options
     targets=(cpu gpu)
-    compilers=(cray claw-cray pgi claw-pgi gnu)
-    fcompiler_cmds=(ftn)
+    compilers=(pgi claw-pgi gnu)
+    fcompiler_cmds=(mpif90 pgfortran gfortran)
 
 
-    export BASE_MODULES="craype-haswell"
+    export BASE_MODULES="craype-x86-skylake"
     export NVIDIA_CUDA_ARCH="sm_70"
 
     # BOOST
     export Boost_NO_SYSTEM_PATHS=true
     export Boost_NO_BOOST_CMAKE=true
 
-    export BOOST_ROOT=/project/c14/install/kesch/boost/boost_1_67_0/
+    export BOOST_ROOT=/project/c14/install/tsa/boost/boost_1_67_0/
```

Comment:
Is this path planned to be made? Because it doesn't exist yet!

Reply:
All libraries/tools should be installed in the new `install/tsa` dir (using jenkins) as far as I understand. I don't know if boost is installed automatically or someone would have to do it manually.

Reply:
Ok, it's fine then, was just wondering if you planned to merge before the path was made, but shouldn't be the case if I understand well.